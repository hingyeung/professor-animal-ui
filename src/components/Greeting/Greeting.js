import React from 'react';

const Greeting = function(props) {
  return (
    <h1>Click Load to load an existing animal definition.</h1>
  )
};

export default Greeting;