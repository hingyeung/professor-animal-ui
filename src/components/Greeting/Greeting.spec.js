import React from 'react';
import renderer from 'react-test-renderer';
import Greeting from "./Greeting";

describe('Greeting', function () {
  it('should render correctly', () => {
    const tree = renderer
    .create(<Greeting/>)
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
