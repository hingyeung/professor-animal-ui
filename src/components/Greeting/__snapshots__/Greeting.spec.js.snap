// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Greeting should render correctly 1`] = `
<h1>
  Click Load to load an existing animal definition.
</h1>
`;
